pyanaconda/modules/storage/bootloader/utils.py

```diff
@@ -215,6 +217,52 @@ def install_boot_loader(storage):
     storage.bootloader.write()
 
 
+def create_bls_entries(sysroot, storage, kernel_versions):
+    """Create BLS entries.
+
+    :param sysroot: a path to the root of the installed system
+    :param storage: an instance of the storage
+    :param kernel_versions: a list of kernel versions
+    """
+    # Not using BLS configuration, skip it
```

Comment:
I think this docstring could be a bit extended to why this condition works - eq. why presence of that binary indicates BLS is not in use.

Reply:
I don't know that.